airflow/ui/src/layouts/Nav.tsx

```diff
@@ -94,7 +70,11 @@ export const Nav = () => {
           <Icon as={AirflowPin} height="35px" width="35px" />
         </Box>
         <NavButton title="Home" icon={<FiHome size="1.75rem" />} isDisabled />
-        <NavButton title="DAGs" icon={<DagIcon height={7} width={7} />} />
+        <NavButton
+          title="DAGs"
+          to="dags"
+          icon={<DagIcon height={7} width={7} />}
+        />
```

Comment:
Maybe not in this PR, but it would be nice to put the navItems in an array and then just map on this component.